docs/relational-databases/track-changes/known-issues-and-errors-change-data-capture.md

```diff
@@ -159,6 +159,12 @@ These are the different troubleshooting categories included in this section:
 
 * **Recommendation**: Ensure the `cdc` user has the `db_owner` role assigned. To create the `cdc` user, see the example [Create cdc user and assign role](#create-user-and-assign-role).
 
+#### Error 22830 - Could not update the metadata that indicates database <database name> is enabled for Change Data Capture. The failure occurred when executing the command 'sp_cdc_create_functions'. 
+
+* **Cause**: presence of a trigger that is inserting data in master database on objects creation.  Enabling cdc is done by a temporary cdc user, so  sp_cdc_create_functions will be executed under this cdc user, the trigger will go off and try to insert data in master database (under the same cdc user) and then this error appears because cdc user does not have required permissions on master database.
```

Comment:
```suggestion
* **Cause**:  This error occurs when a 'CREATE OBJECT' trigger exists in the database and a user that does not have permission to write data to the `master` database tries to enable change data capture (CDC). The `sp_cdc_create_functions` stored procedure is automatically executed by the user when trying to enable CDC, which fires the trigger. Since the user doesn't have permission to write to the `master` database, the trigger fails, which causes the CDC stored procedure to fail with error 22830. 
```